flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterModelRename.java

```diff
@@ -51,12 +51,12 @@ public String[] fullNewModelName() {
 
     @Override
     public List<SqlNode> getOperandList() {
-        return List.of(modelName, newModelName);
+        return List.of(getName(), newModelName);
```

Comment:
nit: we could also just use protected for `name` then `getName` could return String or will simply not be needed anymore.